selenium/che-selenium-core/src/main/java/org/eclipse/che/selenium/core/client/TestGitHubRepository.java

```diff
@@ -135,9 +135,7 @@ private void createFile(Path pathToRootContentDirectory, Path pathToFile) throws
    * @throws IOException
    */
   public void changeFileContent(String pathToFile, String content) throws IOException {
-    ghRepo
-        .getFileContent(String.format("/%s", pathToFile))
-        .update(content, "change - " + pathToFile);
+    changeFileContent(pathToFile, content, format("Change file-%s", pathToFile));
```

Comment:
it seems there is redundant "-" in commit message which makes commit message misleading in part of file path.